README.md

```diff
@@ -29,17 +32,143 @@ System (CSMS) via WebSocket protocol. At the moment only OCPP 2.0 (download [her
 - TransactionEvent
 - StatusNotification
 - Reset (only IMMEDIATE reset is supported)
-- GetVariables (will return REJECTED status)
-- SetVariables (will return ACCEPTED, but does not store variables anywhere)
+- GetVariables
+- SetVariables (heartbeat only)
```

Comment:
I'd say that we support SetVariables message as such, don't see why it's worth mentioning 'heartbeat only', it's a matter of station having or not having specific variable in place. In this case GetVariables are also not fully supported